### DISCUSSION TITLE
Inquiry of providers radios split 7.2 📻

### DISCUSSION THREAD

**Discussion by MohammadYAmmar** [srsran/srsRAN_Project#610] [General Help] [answered]
Hello srsRAN users,

Can anyone provide me list of radios use split 7.2 compatible with srsRAN like Benetel.

Any help will be appreciated and I hope this helps others too!

**Comment by pgawlowicz** [comment, marked as answer]
check the list of the RUs here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/hw_integration.html
